third_party/ijar/ijar.cc

```diff
@@ -368,9 +370,12 @@ static void OpenFilesAndProcessJar(const char *file_out, const char *file_in,
             strerror(errno));
     abort();
   }
-  u8 output_length =
-      in->CalculateOutputLength() +
-      EstimateManifestOutputSize(target_label, injecting_rule_kind);
+  u8 output_length = in->CalculateOutputLength();
+  if (output_length < 103ull + DUMMY_PATH_LENGTH) {
```

Comment:
Where does `103` come from?

Reply:
From measurement, error message tells 108 bytes are needed. "dummy" appears twice in the output, so 98+2*strlen("dummy") should work even if somebody changes "dummy". (I fixed 103).

Somehow I cannot get the same number by adding together numbers used EstimateManifestOutputSize. And I consider reverse engineering zip code and figuring out header sizes a bit too much effort for this fix.